e2e/framework/transport/k8s_transport_client.go

```diff
@@ -0,0 +1,69 @@
+package transport
+
+import (
+	"github.com/rook/rook/e2e/framework/objects"
+	"github.com/rook/rook/e2e/framework/utils"
+)
+
+type k8sTransportClient struct {
+}
+
+func CreateNewk8sTransportClient() *k8sTransportClient {
+	return &k8sTransportClient{}
+}
+
+const (
+	kubectl_executable = "kubectl"
+)
+
+func (k *k8sTransportClient) ExecuteCmd(cmd []string) (stdout string, stderr string, err error) {
+	return utils.ExecuteCmdAndLogToConsole(kubectl_executable, cmd, []string{})
+
+}
+
+func (k *k8sTransportClient) Apply(cmdArgs []string) (stdout string, stderr string, err error) {
+	initialArgs := []string{"replace", "--force", "-f"}
+	cmdArgs = append(initialArgs, cmdArgs...)
+	return utils.ExecuteCmdAndLogToConsole("kubectl", cmdArgs, []string{})
+}
+
+func (k *k8sTransportClient) Execute(cmdArgs []string, optional []string) (stdout string, stderr string, exitCode int) {
```

Comment:
we are going to refactor this in future iterations, we have issues open in GitHub